### PR TITLE
Fixed duplication image and removed unnecessary image links

### PR DESCRIPTION
**Pull request by MichalMaler**
"Pick a section (e.g. End-user Guide), go through all of its content, and fix screenshots. By fixing, I mean to make sure that big images are clickable links to themselves in orig. size, so they can be read. And small images (those that are not automatically resized in the browser) are not clickable so that it's not confusing."

### CODE REVIEW

**Review by slemeur — APPROVED** [review]

File: src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc [hunk 1]
```diff
@@ -176,10 +176,10 @@ image::workspaces/devfile-tab.png[link="{imagesdir}/workspaces/devfile-tab.png"]
 The following is a comparison between workspace configuration commands in Che 6 (Figure 1) and Che 7 (Figure 2):
 
 .An example of the Workspace configuration commands in Che 6
-image::workspaces/workspace-devfile.png[link="{imagesdir}/workspaces/workspace-devfile.png"]
+image::che-6-workspace-confg.png[]
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc]:
```suggestion
image::workspaces/che-6-workspace-confg.png[]
```

Comment (rkratky) [reply src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc]: @MichalMaler The name of the image file should be fixed, too.

File: src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc [hunk 2]
```diff
@@ -100,7 +100,7 @@ To use separate recipe files, the paths can be relative or absolute. For example
 ====
 * Alternatively, add the content as `referenceContent` (the `referenceContent` field replaces the Che 6 recipe content).
 +
-image::workspaces/che-6-workspace-confg.png[link="{imagesdir}/workspaces/che-6-workspace-confg.png"]
+image::workspaces/che-6-workspace-confg.png[]
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc]: Fix file name.

File: src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc [hunk 3]
```diff
@@ -134,9 +134,9 @@ image::workspaces/component_as_a_referenceContent.png[link="{imagesdir}/workspac
 
 . Copy the required fields from the old workspace (`image`, `volumes`, `endpoints`). For example:
 +
-image::workspaces/workspace_config_Che_6.png[link="{imagesdir}/workspaces/workspace_config_Che_6.png"]
+image::workspaces/workspace_config_Che_6.png[]
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc]: No caps in file names.

File: src/main/pages/che-7/end-user-guide/assembly_converting-a-che-6-workspace-to-a-che-7-devfile.adoc [hunk 4]
```diff
@@ -15,10 +15,10 @@ This section describes how to manually convert an old Che 6 workspace configurat
 The following is a comparison between a *Che 7 devfile* (Figure 1) and a *Che 6 workspace configuration* (Figure 2):
 
 .Che 7 referenceContent (devfile)
-image::workspaces/workspace-devfile.png[link="{imagesdir}/workspaces/workspace-devfile.png"]
+image::workspaces/workspace-devfile.png[]
 
 .Che 6 recipe content (workspace configuration)
-image::workspaces/che-6-workspace-confg.png[link="{imagesdir}/workspaces/che-6-workspace-confg.png"]
+image::workspaces/che-6-workspace-confg.png[]
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/assembly_converting-a-che-6-workspace-to-a-che-7-devfile.adoc]: Fix file name.

**Review by rkratky — CHANGES_REQUESTED** [review]

**Review by rkratky — APPROVED** [review]

**Pull request merged**